backend/experiment/models.py

```diff
@@ -462,10 +457,6 @@ def get_fallback_content(self) -> "BlockTranslatedContent":
         Returns:
             Fallback content
         """
-
-        if not self.phase or self.phase.experiment:
-            return self.translated_contents.first()
```

Comment:
We don't have blocks which aren't assigned to phases, and no phases which aren't assigned to experiments anymore. If that happens, I think it's better to throw an error.